x/oracle/keeper/query_no_stake_report.go

```diff
@@ -52,6 +65,12 @@ func (q Querier) GetReportersNoStakeReports(ctx context.Context, req *types.Quer
 		}
 		reports = append(reports, &stringReport)
 		if req.Pagination != nil && uint64(len(reports)) >= req.Pagination.Limit {
+			buffer := make([]byte, pairKeyCodec.Size(pk))
+			_, err = pairKeyCodec.Encode(buffer, pk)
+			if err != nil {
+				return nil, status.Error(codes.Internal, "failed to encode pagination key")
+			}
+			pageRes.NextKey = buffer
```

Comment:
isn't this the current key instead of the next key? should it be moved to the top?